app/models/github-event.js

```diff
@@ -0,0 +1,19 @@
+import Model from 'ember-data/model';
+import attr from 'ember-data/attr';
+import { computed, get } from '@ember/object';
+
+export default Model.extend({
+  action: attr(),
+  eventType: attr(),
+  failureReason: attr(),
+  githubDeliveryId: attr(),
+  insertedAt: attr('date'),
+  payload: attr(),
+  status: attr(),
+  updatedAt: attr('date'),
+
+  prettyPayload: computed('payload', function() {
+    let payload = get(this, 'payload');
+    return JSON.stringify(payload, null, 2);
+  })
```

Comment:
I know our thoughts on this probably diverge, but I dislike anything other than pure attributes and relationships on models, unless it significantly dries up the code. In this specific case, a single controller is the only one using `prettyPayload`, so it would make sense to me to have the property there instead of the model.

It's not a deal breaker for me, but it is my preference. I also don't think it matters internally, performance-wise, since a computed generally does not get evaluated unless used directly or indirectly in a template.